opdracht4.js

```diff
@@ -0,0 +1,65 @@
+// Opdracht 4a: Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft in het format [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV.
+//
+function showTvName(i) {
+  return (
+    inventory[i].brand + " " + inventory[i].type + " - " + inventory[i].name
+  );
+}
+
+//     Opdracht 4b: Maak een herbruikbare functie die de prijs van één tv als parameter verwacht (zoals 379 of 159) teruggeeft in het format €379,- of €159,-.
+function showPrice(i) {
+  const price = inventory[i].price;
+  return "€" + price + ",-";
+}
+
+// Opdracht 4c: Maak een herbruikbare functie die een string genereert voor alle beschikbare schermgroottes van één tv. De functie geeft dit terug in het format [schermgrootte] inches ([schermgrootte omgerekend]cm) | [schermgrootte] inches ([schermgrootte omgerekend]cm) etc. Als een tv maar één schermgrootte heeft ([32]) wordt de output 32 inch (81 cm). Wanneer een tv vier schermgroottes heeft ([43, 50, 55, 58]) wordt de output 43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm). Let op: om één string te genereren uit een array van schermgroottes zul je een for-loop voor moeten gebruiken.
+function showTvScreenSizes(i) {
+  const allSizes = inventory[i].availableSizes;
+  let output = "";
+  const iToCm = 2.54;
+  if (allSizes.length === 1) {
+    const sizeInCm = allSizes * iToCm;
+    output = allSizes + " inch (" + Math.floor(sizeInCm) + " cm)";
+  } else {
+    for (let i = 0; i < allSizes.length; i++) {
+      const sizeInCm = allSizes[i] * iToCm;
+      output += allSizes[i] + " inch (" + Math.floor(sizeInCm) + " cm) | ";
+    }
```

Comment:
```suggestion
  for (let i = 0; i < allSizes.length; i++) {
      const sizeInCm = allSizes[i] * iToCm;
      output += allSizes[i] + " inch (" + Math.floor(sizeInCm) + " cm) | ";
  }
```

Dit zou als het goed is hetzelfde resultaat moeten hebben